src/multi_physics/QED/src/physics/breit_wheeler/breit_wheeler_engine_tables.hpp

```diff
@@ -0,0 +1,828 @@
+            m_params{params},
+            m_table{containers::equispaced_1d_table<RealType, VectorType>{
+                    math::m_log(params.chi_phot_min),
+                    math::m_log(params.chi_phot_max),
+                    vals}}
+            {
+                m_init_flag = true;
+            };
+
+            /*
+            * Generates the content of the lookup table (not usable on GPUs).
+            * This function is implemented elsewhere
+            * (in breit_wheeler_engine_tables_generator.hpp)
+            * since it requires a recent version of the Boost library.
+            *
+            * @tparam Policy the generation policy (can force calculations in double precision)
+            *
+            * @param[in] show_progress if true a progress bar is shown
+            */
+            template <generation_policy Policy = generation_policy::regular>
+            void generate(const bool show_progress  = true);
+
+            /*
+            * Initializes the lookup table from a byte array.
+            * This method is not usable on GPUs.
+            *
+            * @param[in] raw_data the byte array
+            */
+            dndt_lookup_table(std::vector<char>& raw_data)
+            {
+                using namespace utils;
+
+                constexpr size_t min_size =
+                    sizeof(char)+ //single or double precision
+                    sizeof(m_params);
+
+                if (raw_data.size() < min_size)
+                    throw std::runtime_error("Binary data is too small to be a Breit Wheeler \
+                     T-function lookup-table.");
+
+                auto it_raw_data = raw_data.begin();
+
+                if (serialization::get_out<char>(it_raw_data) !=
+                    static_cast<char>(sizeof(RealType))){
+                    throw std::runtime_error("Mismatch between RealType used to write and to read \
+                        the Breit Wheeler T-function lookup-table");
+                }
+
+                m_params = serialization::get_out<
+                    dndt_lookup_table_params<RealType>>(it_raw_data);
+                m_table = containers::equispaced_1d_table<
+                    RealType, VectorType>{std::vector<char>(it_raw_data,
+                        raw_data.end())};
+
+                m_init_flag = true;
+            };
+
+            /**
+            * Operator==
+            *
+            * @param[in] rhs a structure of the same type
+            *
+            * @return true if rhs is equal to *this. false otherwise
+            */
+            PXRMP_INTERNAL_GPU_DECORATOR PXRMP_INTERNAL_FORCE_INLINE_DECORATOR
+            bool operator== (
+                const dndt_lookup_table<
+                    RealType, VectorType> &b) const
+            {
+                return
+                    (m_params == b.m_params) &&
+                    (m_init_flag == b.m_init_flag) &&
+                    (m_table == b.m_table);
```

Comment:
```suggestion
                    RealType, VectorType> &rhs) const
            {
                return
                    (m_params == rhs.m_params) &&
                    (m_init_flag == rhs.m_init_flag) &&
                    (m_table == rhs.m_table);
```

Maybe rename the input parameter `rhs` to remain consistent with the other operators `==` and with the doxygen comments?